Print classifier info

jo-mueller opened the pull request:
Hi @haesleinhuepf ,

here comes a clean version of PR #29 
this should fix #23  - I also added @kevinyamauchi 's suggestion to put the printed classifier informations int the `__str__` and `__repr__` methods.

Let me know what you think 👍

haesleinhuepf commented:
Hi Johannes @jo-mueller ,

that looks great! Could you please check if it works for all classifiers? Maybe also add some test-code for the other classifiers.

Thanks!

Best,
Robert

jo-mueller commented:
>Could you please check if it works for all classifiers? Maybe also add some test-code for the other classifiers.

Thanks for pointing this out, I didn't check that the other classifiers besides `PixelSegmenter` and `ObjectSegmenter` were not inherited from `PixelClassifier` and thus couldn't use the parent class' `info()` function. I added the missing functions to the Downstream classifiers (`ObjectClassifer` & `TableRowSegmenter`) and added tests for all all classifier types.

haesleinhuepf commented:
Awesome, thanks Johannes @jo-mueller !
